Source/Core/Core/Debugger/PPCDebugInterface.cpp

```diff
@@ -324,6 +325,40 @@ std::string PPCDebugInterface::GetDescription(unsigned int address)
   return g_symbolDB.GetDescription(address);
 }
 
+u32 PPCDebugInterface::GetMemoryAddressFromInstruction(std::string instruction)
+{
+  int offs = 0;
```

Comment:
```suggestion
  u32 offset = 0;
```
(and handle necessary conversions to u32 explicitly). Otherwise expressions further down are intermixing signed and unsigned types implicitly.

Reply:
offset might be negative (-0x6000 rtoc). roffs is always a memory address.  Do I use long long for everything and convert to u32 at the end? Or is there a trick I'm missing.